src/2023/index.html

```diff
@@ -4,6 +4,16 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Helvetic Ruby - A Ruby conference in Switzerland">
+    <meta name="author" content="Helvetic Ruby">
+    <meta property="og:title" content="Helvetic Ruby - A Ruby conference in Switzerland">
+    <meta property="pg:description" content="A single-day, single-track conference for Ruby enthusiasts and professionals from Switzerland and abroad.">
```

Comment:
There's a typo in the `og:description` field here in all other files:

```suggestion
    <meta property="og:description" content="A single-day, single-track conference for Ruby enthusiasts and professionals from Switzerland and abroad.">
```